tier2/{{cookiecutter.project_slug}}/repometrics/hooks/post_gen_project.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
Can this be a bash script instead of a Python script?

Reply:
I finished the post-gen python script, I can give this a look!